build/aca-machine-init.sh

```diff
@@ -97,28 +100,24 @@ echo "6--- installing openvswitch dependancies ---" && \
     make && \
     make install && \
     cp /var/local/git/openvswitch/lib/vconn-provider.h /usr/local/include/openvswitch/vconn-provider.h && \
-    cd /usr/local/bin && \
-    rm ov* && \
-    rm -rf \var\local\git\openvswitch
+    rm -rf /var/local/git/openvswitch
     cd ~
 
 echo "7--- building alcor-control-agent"
-cd ~/alcor-control-agent && cmake . && make
+cd $BUILD/.. && cmake . && make
 
 echo "8--- rebuilding br-tun and br-int"
-ovs-vsctl add-br br-int
-ovs-vsctl add-br br-tun
-ovs-vsctl add-port br-int patch-tun
-
-ovs-vsctl set interface patch-tun type=patch
-ovs-vsctl set interface patch-tun options:peer=patch-int
-ovs-vsctl add-port br-tun patch-int 
-ovs-vsctl set interface patch-int type=patch
-ovs-vsctl set interface patch-int options:peer=patch-tun
+ovs-ctl --system-id=random --delete-bridges restart
+ovs-vsctl add-br br-int -- add-br br-tun
+ovs-vsctl \
```

Comment:
nice optimizations :)

Reply:
merge commands as one also avoid device not exist error in ovsDB caused by command sequence